app/models/user.rb

```diff
@@ -157,7 +157,7 @@ class User < ApplicationRecord
   has_many :notification_settings, dependent: :destroy
   has_many :one_signal_players, dependent: :destroy
   has_many :reposts, dependent: :destroy
-
+  has_many :user_ipaddresses, dependent: :destroy
```

Comment:
`has_many :ip_addresses, dependent: :destroy, class_name: 'UserIpAddress'` is probably nicer